TodoList/TodoList/Controllers/UserController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using TodoList.Data;
+using TodoList.DTOs;
+using TodoList.Service;
+namespace TodoList.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserController : ControllerBase
+    {
+        private readonly ILogger _logger;
+        private readonly IUsers _users;
+
+        public UserController(ILogger<UserController> logger, IUsers users)
+        {
+            _logger = logger;
+            _users = users;
+        }
+
+        [HttpPost("SignUp")]
```

Comment:
```suggestion
        [HttpPost()]
```
xem lại qui tắc đặt tên path khi tạo 1 record trong DB nhé